test/mlir/driver/unix/check_invalid_suffix_case.mlir

```diff
@@ -0,0 +1,4 @@
+// RUN: (onnx-mlir input.afo || exit 0) 2>&1 | FileCheck %s
+
+// REQUIRES: system-linux
```

Comment:
Same for all of the tests, really.

Reply:
Removed the requirements for all other tests. Thank you!